.github/dependabot.yml

```
@@ -5,3 +5,10 @@ updates:
     schedule:
       interval: "daily"
     labels: ["dependencies"]
+  # Dependabot only updates hashicorp GHAs, external GHAs are managed by internal tooling (tsccr)
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

```suggestion
      interval: "weekly"
```

IMO `daily` is unnecessarily obnoxious and it's nice if these are kept in sync with the config we use for the security-tsccr automation which is also weekly.